css/style.css

```diff
@@ -0,0 +1,41 @@
+#image{
+    display: block;
+    margin:0 auto;
+    max-width: 60%;
+    
```

Comment:
- Make sure to follow [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), so please make sure to use two spaces of indentation.

Reply:
- [ ] You did a good job finding this lack of good practices, keep it up! But I would suggest you try to find different words to try not to sound like ordering. Instead of saying `Make sure to follow...` maybe you could type something like `Kindly check the HTML & CSS practices and make sure to follow them to make your code easier to read and also easier to understand for the next person that needs to take a look at it`. So this way you would sound more like suggesting rather than ordering something.